grammar.js

```diff
@@ -920,9 +922,8 @@ module.exports = grammar(C, {
       $.scoped_namespace_identifier,
     ),
 
-    operator_name: $ => token(seq(
+    operator_name: $ => prec(1, seq(
```

Comment:
If tests are green with this change, I'm fine with it.
Anyway I don't really understand how this change is related to the PR title.
If it's unrelated please create an other PR else my bad.

Reply:
This was addressing Max's suggestion of not using explicit `/\s*/` here. It's not specifically needed for user-defined literals, but since it already requires a change in this rule, a little clean up probably doesn't hurt?

(For some background, I actually added the `/\s*/` when I created this PR, but it seems someone else made the exact same change independently and that was already merged.)

Reply:
Yeah, @calixteman the goal was to break this out into separate tokens, which will be better for syntax highlighting. It's somewhat independent of this PR, but somewhat related, because this PR adds yet another token boundary (`"" my_suffix`) to this rule.